swarm_test.go

```diff
@@ -114,7 +112,7 @@ func TestNewSwarm(t *testing.T) {
 			configure: func(config *api.Config) {
 				config.SwapBackendURL = ipcEndpoint
 				config.SwapEnabled = true
-				config.NetworkID = swap.AllowedNetworkID
+				config.NetworkID = 2
```

Comment:
better use constant still, so that the name gives away why we expect that value to behave the way we want

Reply:
Changed